pkg/stream/operators.go

```diff
@@ -193,6 +195,7 @@ func Debounce[T any](src Observable[T], duration time.Duration) Observable[T] {
 					select {
 					case <-done:
 						complete(ctx.Err())
+						<-errs
```

Comment:
Ah good point. I made this overly complicated. I don't need this case at all since `errs` will get the "context canceled" error from `src` anyway. Will fix.